pkg/docker/config/config_test.go

```diff
@@ -372,20 +421,19 @@ func TestGetAuthFailsOnBadInput(t *testing.T) {
 	configPath := filepath.Join(configDir, "auth.json")
 
 	// no config file present
-	username, password, err := GetAuthentication(nil, "index.docker.io")
+	auth, err := GetCredentials(nil, "index.docker.io")
 	if err != nil {
 		t.Fatalf("got unexpected error: %#+v", err)
 	}
-	if len(username) > 0 || len(password) > 0 {
-		t.Fatalf("got unexpected not empty username/password: %q/%q", username, password)
-	}
+
+	require.Equal(t, types.DockerAuthConfig{}, auth, "got unexpected non empty auth credential: %v", auth)
```

Comment:
- This should be `assert` (if this fails, other parts of the test function can continue).
- Please drop the blank line above
- (Absolutely non-blocking: No need to keep the “got unexpected …” format string, `.Equal` prints something by default, including the unexpected value, IIRC.)

Reply:
> this should be assert (if this fails, other parts of the test function can continue).

The previous behavior was t.Fatal on error condition. Changing to assert per suggestion.

Reply:
I’m sorry, I somehow thought this is a new test. You’re right that this was previously fatal, and unrelated to this PR.